eng/native/configuretools.cmake

```diff
@@ -42,6 +42,7 @@ if(NOT WIN32 AND NOT CLR_CMAKE_TARGET_BROWSER)
   locate_toolchain_exec(ar CMAKE_AR)
```

Comment:
Could you please add quotes around values in the function:

```diff
-       message(FATAL_ERROR "Unable to find toolchain executable. Name: ${exec}, Prefix: ${TOOLSET_PREFIX}.")
+      message(FATAL_ERROR "Unable to find toolchain executable. Name: '${exec}', Prefix: '${TOOLSET_PREFIX}'.")
```
so the error message looks like:

>   Unable to find toolchain executable.  Name: 'objcopy', Prefix: ''.
instead of

>   Unable to find toolchain executable.  Name: objcopy, Prefix: .

Reply:
Sure!